blog/_posts/2015-10-03-this-isnt-a-goodbye-it-is-a-continuity.md

```diff
@@ -0,0 +1,135 @@
+
+During this time, we have studied, investigated, read, attended courses and workshops, talks, we have done hundreds of questions and received hundreds of answers or explanations, we have practiced and written many code lines. We have improved our knowledge about JavaScript, Ruby, HTML, CSS, Git and Markdown, we have taken our first steps in Rails and we have learned to use Sinatra, SASS, SCSS, HAML, Jekyll, RSPEC, Minitest,  as well as the composition and performance of the gems and, in depth, about the structure and logic of the gem which we are working on: [Living Style Guide](http://livingstyleguide.org/).  
+
+In our first blog post, we already said that since the first moment we believed in the potentiality of this gem bit by bit, as we have been analyzing its code, searched in order to do our task and complete it, we have confirmed  and, in point of fact, it has allowed us grow and learned so much about different areas of programming and also, it has turned us aware of what we have been and are able to do.
+There have been hundreds of learning experiences, but the major revelation has been programming as profession and way of life.  
+
+### OUR PROJECT: LIVING STYLE GUIDE  
+
+Ended our summer of code, we can say that we worked and we completed the tasks that we had proposed and that had been accorded with our mentor Nico. Some of them were more simple and some were more complex that took us more time, effort and study than we expected. But, this was precisely what we wanted it: to grow with the project.  
+
+#### What functionalities have we included in the gem?  
+
+- We included the functionality to be able copy the hexadecimal codes  or the variables in the colors examples through a Click event or Alt + Click with vanilla javascript.  
+
+- We worked on a button that allows copying  your style guide code blocks.  
+
+- We added a search-box that allows searching any word inside of your style guide. This search-box is a filter, so that the user of the gem can decide whether to put it or not in his style guide, putting @search-box, as well as personalizing its placeholder:  
+
+          @search-box placeholder: I am search …  
+
+- We did a button to hide the code blocks that appear in your style guide, and same as the former one, this is a filter that can be included or not, depending on the user:  
+
+          @toggle-code  
+
+- We included a menu of navigation that  is automatically generated  with the style guide in function of the titles and subtitles that the user includes in his own. This means, that is personalized and suitable to the style of each user. ( in process of getting merged)  
```

Comment:
`each user. ( in process of getting merged)  `
`each user (in process of getting merged).`